src/index.html

```diff
@@ -5,8 +5,375 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    > 
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      value="crossorigin"
+    > 
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet"
+    >  
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="header__block">
+        <ul class="header__links">
+          <li>
+            <a href="#1" class="is-active">Apple
```

Comment:
let's make nav a separate block

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#1" class="nav__link is-active">Apple
```